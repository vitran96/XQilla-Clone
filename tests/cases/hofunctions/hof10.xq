for $f in (concat("one ", ?, " three"), substring-before("one two three", ?), matches(?, "t.*o"), xs:NCName(?))
return <result>{$f("two")}</result>
