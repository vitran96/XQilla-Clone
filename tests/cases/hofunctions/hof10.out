<result>one two three</result><result>one </result><result>true</result><result>two</result>